src/sql/src/plan/statement/ddl.rs

```diff
@@ -1059,18 +1043,44 @@ pub fn plan_create_subsource(
 generate_extracted_config!(LoadGeneratorOption, (TickInterval, Interval));
 
 pub(crate) fn load_generator_ast_to_generator(
-    generator: &mz_sql_parser::ast::LoadGenerator,
+    loadgen: &mz_sql_parser::ast::LoadGenerator,
     _options: &[LoadGeneratorOption<Aug>],
-) -> Result<mz_storage::types::sources::LoadGenerator, PlanError> {
-    let load_generator = match generator {
+) -> Result<
+    (
+        mz_storage::types::sources::LoadGenerator,
+        Option<HashMap<FullObjectName, (usize, RelationDesc)>>,
+    ),
+    PlanError,
+> {
+    let load_generator = match loadgen {
         mz_sql_parser::ast::LoadGenerator::Auction => {
             mz_storage::types::sources::LoadGenerator::Auction
         }
         mz_sql_parser::ast::LoadGenerator::Counter => {
             mz_storage::types::sources::LoadGenerator::Counter
         }
     };
-    Ok(load_generator)
+
+    let mut available_subsources = HashMap::new();
+    let generator = as_generator(&load_generator);
+    for (i, (name, desc)) in generator.views().iter().enumerate() {
+        let name = FullObjectName {
+            database: RawDatabaseSpecifier::Name("mz_loadgenerator".to_owned()),
+            schema: loadgen.to_string().to_lowercase(),
```

Comment:
If you happen to go back through CI:

```suggestion
            database: RawDatabaseSpecifier::Name("mz_load_generators".to_owned()),
            schema: match loadgen {
                LoadGenerator::Counter => "counter".into(),
                LoadGenerator::Auction => "auction.into(),
                // Please use `snake_case` for any multi-word load generators
                // that you add.
            },
```

If not I'll send it up as a follow on.

Reply:
Added.